builtin_interfaces/Time stamp
int16 team_a_score
int16 team_b_score

float32 puck_x
float32 puck_y
float32 puck_vx
float32 puck_vy

float32[] agent_x
float32[] agent_y
